src/posts/posts.controller.spec.ts

```diff
@@ -20,8 +27,14 @@ describe('PostsController', () => {
   });
 
   describe('createPost function', () => {
-    it('should throw not implemented', () => {
-      expect(controller.createPost).toThrowError(new NotImplementedException());
+    it('should return a string', () => {
```

Comment:
Please test error handling

Reply:
It seems that there is no error case here in posts.controller. As any error will be thrown in the methods inside it by nestjs. Accordingly, error handling tests will be written whenever there is an error might be thrown inside the method under test itself. E.g. inside postsRepository method